src/Events/Events.ts

```diff
@@ -312,6 +451,144 @@ class Events {
         return Requests.post(route, data, query, options);
     }
 
+    public static createGroup (event_id : string, data : object,query? : object | null,options? : object |null  ) {
+        let route = this.routeCreateGroup.route.replaceAll('{id}',event_id);
+        
+        return Requests.post(route,data,query,options);
+    } 
+    public static updateGroup(event_id : string, data : object, query? : object | null, options? : object | null) {
```

Comment:
There should be another parameter, group_id. And the parameter should be replaced in the route as well.

Reply:
How is this resolve? group_id is still missing.